ios/FluentUI/Controls/ButtonVnext.swift

```diff
@@ -168,57 +173,93 @@ public struct MSFButtonViewButtonStyle: ButtonStyle {
 /// View that represents the button
 public struct MSFButtonView: View {
     var action: () -> Void
+    @Environment(\.theme) var theme: FluentUIStyle
     @ObservedObject var tokens: MSFButtonTokens
     @ObservedObject var state: MSFButtonVnextState
 
     public init(action: @escaping () -> Void,
                 style: MSFButtonVnextStyle,
                 size: MSFButtonVnextSize) {
         self.action = action
-        self.tokens = MSFButtonTokens(style: style, size: size)
         self.state = MSFButtonVnextState()
+        self.tokens = MSFButtonTokens(style: style, size: size)
     }
 
     public var body: some View {
         Button(action: action, label: {})
             .buttonStyle(MSFButtonViewButtonStyle(targetButton: self))
             .disabled(state.isDisabled)
             .frame(maxWidth: .infinity)
+            .onAppear {
```

Comment:
Should we use the /// commenting style here?

Reply:
Typically we use /// (Xcode style) for documentation comments decorating classes, methods, attributes, etc...
This is just an informative comment inside the code.